### DISCUSSION TITLE
https://korhonen.cc/fi/pgp/

### DISCUSSION THREAD

**Discussion by giscus** [FunctionalHacker/korhonen.cc#2] [Announcements]
# https://korhonen.cc/fi/pgp/

Tykkään PGP:stä ja turvaan sillä kaiken minkä pystyn. Jos haluaisit lähettää minulle enkryptoituja tiedostoja, sähköposteja tai muuta vastaavaa, tulit oikeaan paikkaan. Seuraavaksi on julkinen avaimeni saatavilla eri formaateissa.
   Sormenjälki 1EFC4DBB079407113A761DF7C514037F1EFF4FA6
 Tämä on avaimeni sormenjälki ja se löytyy monilta avainservereiltä. SKS avainserverien sulkeutumisen jälkeen en ole juurikaan ylläpitänyt avaintani servereillä.
   ASCII Armor Klikkaa tästä
   OpenKeychain Seuraava QR koodi pitäisi olla yhteensopiva OpenKeyChain Android-sovelluksen kanssa.

https://korhonen.cc/fi/pgp/

**Comment by FunctionalHacker** [comment]
Testataan suomenkielinen kommentti myös 🇫🇮
